web_m2x_options/static/src/components/form.esm.js

```diff
@@ -171,11 +171,14 @@ patch(many2OneField, {
         props.fieldColorOptions = options.colors;
         return props;
     },
-    extractProps({attrs, context, decorations, options, string}, dynamicInfo) {
+    extractProps({attrs, context, decorations, options, string, type}, dynamicInfo) {
         const props = super.extractProps(
             {attrs, context, decorations, options, string},
             dynamicInfo
         );
+        if (type === "many2one_reference") {
+            return props;
+        }
```

Comment:
How about solving this more generic?
```suggestion
        if (!this.m2m_options_props){
            return props;
        }
```